ailab/benchmark/views.py

```diff
@@ -73,6 +73,11 @@ def visualize(request):
         qs = BenchmarkResult.objects.filter(result_q)
     else:
         qs = BenchmarkResult.objects.all()
+        filters = {
+            'condition': 'AND',
+            'rules': [{}],
+        }
+        print(filters)
```

Comment:
why do you print filters here?

Reply:
That was for debug--thanks for catching!